doc/options.yaml

```diff
@@ -1463,6 +1463,15 @@ verifyExtra:
     This option is for debugging the adjoint only.
     It is used to verify dIda.
 
+sepSensorModel:
+  desc: >
+    This option allows to pick separation sensor option. The 2 options are "heaviside" and "surfvec".
```

Comment:
Please fix the grammar here. Also, you don't have to say the 2 options when they are listed below anyway